gabrielaEnache/src/main/java/com/endava/store/CompanyOrder.java

```diff
@@ -0,0 +1,22 @@
+package com.endava.store;
+
+import java.util.List;
+
+public class CompanyOrder extends AbstractOrder {
+    private final Company company;
+
+    public CompanyOrder(Company company, List<Item> items) {
+        super(items);
+        this.company = company;
+    }
+
+    public Company getCompany() {
+        return company;
+    }
+
```

Comment:
Please pay attention that you are not using getCompany in Main class. So it is not necessary here, or it should be used in main class

Reply:
Fixed